site/app/Resources/views/base.html.twig

```diff
@@ -134,7 +135,7 @@
   </footer>
 
   {% block javascripts %}
-    {% javascripts filter="?jsqueeze" output="js/app_body.js" '@js_body_end' %}
+    {% javascripts filter="?jsqueeze" output="js/app_body.js" '@js_body_end' 'assets/js/bootstrap-select.min.js' %}
```

Comment:
Should this be present in every page load?

I would rather prefer it to be included in `ProjectRepoBundle::Project::new.html.twig`

Reply:
yes, please make it only for the "new project" page.